docs/Developers/Newcomers_Guide.rst

```diff
@@ -23,6 +23,8 @@ following tasks, you will become a developer at coala:
   - merge a ``difficulty/low`` Pull Request
   - review at least a ``difficulty/low`` or higher Pull Request
 
+Note: please merge only *one* ``difficulty/newcomer`` Pull Request
```

Comment:
I have a little suggestion. I think P should be capital in **please** . I think anyone else should also have a look. Rest looks good.